STIG overlay XCCDF shall include Benchmark/@id

isimluk opened the pull request:
otherwise we produce invalid XCCDF.

Addressing:
```
File 'rhel7/unlinked-stig-xccdf.xml' line 2: Element '{http://checklists.nist.gov/xccdf/1.1}Benchmark': The attribute 'id' is required but missing.
```

jan-cerny commented:
@isimluk  Great patch! I tried to validate the generated file and I still get some errors. Can you take a look at them?

```
[jcerny@thinkpad build{overlay-fixes}]$ oscap xccdf validate rhel7/unlinked-stig-xccdf.xml 
File 'rhel7/unlinked-stig-xccdf.xml' line 3: Element '{http://checklists.nist.gov/xccdf/1.1}title': This element is not expected. Expected is ( {http://checklists.nist.gov/xccdf/1.1}status ).
OpenSCAP Error: Invalid XCCDF Checklist (1.1) content in rhel7/unlinked-stig-xccdf.xml. [/builddir/build/BUILD/openscap-1.3.1/src/source/oscap_source.c:346]
```

dahaic commented:
Just curious - is build artifact meant to be valid XCCDF? Should we care?

isimluk commented:
Guys, feel free to merge this PR as it is or close it as it is.

To shed some more light on this. I came across the failures and fixed the first one. However, I changed my path since then, so I am no longer on path to fix all of the failures.

jan-cerny commented:
@isimluk great, so let's merge it, because it's definitely an improvement

@dahaic I have no idea. It's true that some of the files in the build/<product> directories are invalid because they're interim files, but I don't know what is the intention of this particular file.

shawndwells commented:
> Just curious - is build artifact meant to be valid XCCDF? Should we care?

Nah. They're just incremental pieces.

Someday the XSLT can be revisited to make the whole combination process cleaner.
